examples/common.cpp

```diff
@@ -13,6 +13,10 @@
 #include <alloca.h>
 #endif
 
+#if defined(_WIN32) || defined(_WIN64)
+#include <windows.h>
```

Comment:
Just move this include below into the `_WIN32` block.

`_WIN32` is always defined on Windows (even for on 64-bits), so there is no need to check for `_WIN64`